python/shotgun_data/shotgun_data_retriever.py

```diff
@@ -1068,25 +1068,32 @@ def _on_task_completed(self, task_id, group, result):
             return
 
         action = result.get("action")
-        if action in ["find", "find_one", "create", "delete", "update"]:
-            self.work_completed.emit(str(task_id), action, {"sg":result["sg_result"]})
-        elif action == "nav_expand":
-            self.work_completed.emit(str(task_id), "nav_expand", {"nav":result["nav_result"]})
+        if action in ["find", "find_one", "create", "delete", "update", "nav_expand"]:
```

Comment:
ah! yes - good point, i wanted to bring this up and forgot (sorry code was written over an extended period of time). The nav data is brand new and not used anywhere. We do run the risk of breaking things but it's very low. This `nav` key makes things real hard - it's not how it was intended to be used - the `sg` key denotes that you are getting data back from Shotgun in some form or another - each action denotes what type of data is to be expected (find, create, delete, nav etc). I wish we could change the schema one too.